zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -557,6 +561,26 @@ impl<F: Field> MPTConfig<F> {
                     (FixedTableTag::RangeKeyLen256, 256),
                     (FixedTableTag::RangeKeyLen16, 16),
                 ] {
+                    // When using two byte range check, we ensure the tuple within 
+                    // ([0..255], [0..255], [-67,.. odd numbers .., 67]) 
+                    // where n is the possible sum of any consecutive indexs of the first and second byte
+                    if self.two_bytes_lookup {
+                        for n in -max_length..=max_length {
+                            let range = if n <= 0 && range == 256 { 1 } else { range };
+                            for idx1 in 0..range {
+                                for idx2 in 0..range {       
```

Comment:
Do we have to be that precise? I think this should be just a sanity checks at load time before going into RLP checks in other logical gadgets, like the RLP limitation.

Reply:
Yep it needs to be precise! Because the rlc encoding will assume it's `0` for all values after `len` (if we don't check itt the prover can manipulate the data in whichever way he wishes and mess up the rlc value).

Reply:
I see your point, then my summation of first and second idx actually comes in handy :)
When the sum == 0, we're at the cross over of range 225 and 0, and when sum < 0 we're at 0 territory. You think this work?

Reply:
well i just realized that with the sum, just use the idx of the first byte does the same thing, so i'll just use that